src/main/java/com/thevoxelbox/voxelsniper/brush/type/TreeSnipeBrush.java

```diff
@@ -10,36 +10,66 @@
 import com.thevoxelbox.voxelsniper.util.material.Materials;
 import org.bukkit.ChatColor;
 
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
+import java.util.stream.Stream;
 
 public class TreeSnipeBrush extends AbstractBrush {
```

Comment:
See my comment on generate tree, it may apply here too
![](https://i.imgur.com/pbSRSfA.png)